src/views/tables/PositionsTable/PositionsTriggersCell.tsx

```diff
@@ -12,88 +17,144 @@ import { layoutMixins } from '@/styles/layoutMixins';
 import { Button } from '@/components/Button';
 import { Icon, IconName } from '@/components/Icon';
 import { Output, OutputType } from '@/components/Output';
+import { WithHovercard } from '@/components/WithHovercard';
 
-import { isMarketOrderType } from '@/lib/orders';
-
-export type PositionTableConditionalOrder = {
-  price: number;
-  size: number;
-  triggerPrice: Nullable<number>;
-  timeInForce: Nullable<AbacusOrderTimeInForces>;
-  type: AbacusOrderTypes;
-};
+import { isMarketOrderType, isStopLossOrder } from '@/lib/orders';
 
 type ElementProps = {
   market: string;
-  stopLossOrders: PositionTableConditionalOrder[];
-  takeProfitOrders: PositionTableConditionalOrder[];
+  liquidationPrice: Nullable<number>;
+  stopLossOrders: SubaccountOrder[];
+  takeProfitOrders: SubaccountOrder[];
   onViewOrdersClick: (market: string) => void;
-  positionSize?: number | null;
+  positionSide: Nullable<AbacusPositionSides>;
+  positionSize: Nullable<number>;
   isDisabled?: boolean;
 };
 
 export const PositionsTriggersCell = ({
   market,
+  liquidationPrice,
   stopLossOrders,
   takeProfitOrders,
   onViewOrdersClick,
+  positionSide,
   positionSize,
   isDisabled, // TODO: CT-656 Disable onViewOrdersClick behavior when isDisabled
 }: ElementProps) => {
   const stringGetter = useStringGetter();
 
-  const renderOutput = ({
-    label,
-    orders,
-  }: {
-    label: string;
-    orders: PositionTableConditionalOrder[];
-  }) => {
-    const triggerLabel = (warning: boolean = false) => (
-      <Styled.Label warning={warning}>{label}</Styled.Label>
-    );
-    const viewOrdersButton = (
-      <Styled.Button
-        type={ButtonType.Link}
-        action={ButtonAction.Navigation}
-        size={ButtonSize.XSmall}
-        onClick={() => onViewOrdersClick(market)}
-      >
-        {stringGetter({ key: STRING_KEYS.VIEW_ORDERS })}
-        {<Styled.ArrowIcon iconName={IconName.Arrow} />}
-      </Styled.Button>
-    );
+  const showLiquidationWarning = (order: SubaccountOrder) => {
+    if (!isStopLossOrder(order) || !liquidationPrice) {
+      return false;
+    } else if (
+      (positionSide === AbacusPositionSide.SHORT && order.price > liquidationPrice) ||
+      (positionSide === AbacusPositionSide.LONG && order.price < liquidationPrice)
+    ) {
+      return true;
+    }
```

Comment:
Could we return this conditional instead of the `return true` and `return false` after